lib/src/invariant.dart

```diff
@@ -1,19 +1,17 @@
-import 'dart:developer';
-
 import 'package:turf/turf.dart';
 
 ///
-/// Unwrap a coordinate from a Point Feature, Geometry or a single coordinate.
+/// Unwraps a coordinate from a Point, Feature<Point>, and a Position.
 ///
 /// @name getCoord
-/// @param {Array<number>|Geometry<Point>|Feature<Point>} coord GeoJSON Point or an Array of numbers
-/// @returns {Array<number>} coordinates
+/// @param {Position |Point |Feature<Point>} coord GeoJSON Point or an Array of numbers
```

Comment:
this `@param`, `@name`, `@returns` do not exist in Dart documentation, these  Javascript specific. In Dart you just write continuous text with the types annotated in square brackets.

Here is an example: https://github.com/dartclub/turf_dart/blob/main/lib/src/meta.dart

This is a guide on writing documentation in Dart:
https://dart.dev/guides/language/effective-dart/documentation